go/vt/vttablet/tabletserver/query_executor_test.go

```diff
@@ -1548,6 +1550,7 @@ func initQueryExecutorTestDB(db *fakesqldb.DB) {
 		"varchar|int64"),
 		"Innodb_rows_read|0",
 	))
+	sidecardb.AddVTSchemaInitQueries(db)
```

Comment:
I think it is fine as is rather than requiring tests to know whether they need this or not: initializing this here makes sure all tests have this and if a test doesn't need it just won't use it. We do similar things for example while adding queries to ignore, regardless of which test is using fakesqldb.